tests/std/tests/P0528R3_cmpxchg_pad/test.cpp

```diff
@@ -174,7 +173,8 @@ struct X20 {
 template <class X, std::size_t S>
 void test() {
     static_assert(sizeof(X) == S, "Unexpected size");
-    static_assert(!std::has_unique_object_representations_v<X>, "No padding type");
+    static_assert(
+        !std::has_unique_object_representations_v<X>, "Type has no padding which makes testing for P0528 ineffective");

```

Comment:
This is ambiguous between "(Type has no padding) which makes testing for P0528 ineffective"  - type is bad - and "Type has no (padding which makes testing for P0528 ineffective)" - type is good.
```suggestion
        !std::has_unique_object_representations_v<X>, "Type without padding is not useful for testing P0528.");
```